packages/ckeditor5-ui/docs/framework/guides/deep-dive/focus-tracking.md

```diff
@@ -73,13 +73,13 @@ To spice things up even more, you should also know `isFocused` will change when
 Say, for instance, your application wants to focus an editable area of CKEditor 5 when a certain action is executed (e.g. a button is clicked). To do that, use the {@link module:engine/view/view~View#focus `focus()`} method of the editing view:
```

Comment:
Please, be very careful with the context when making a bulk changes like that.

For instance this line no longer makes sense. It says:

To do that, use the {@link **module:engine/view/view~View#focus \`focus()\`**} method of the **editing view**:

This is no longer relevant after your change.

Reply:
Done, thanks for vigilance 👍🏻